var/spack/repos/builtin/packages/phist/package.py

```diff
@@ -0,0 +1,135 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+import os
+import fnmatch
+from spack import *
+
+
+class Phist(CMakePackage):
+    """ PHIST - a Pipelined, Hybrid-parallel Iterative Solver Toolkit.
+
+PHIST provides implementations of and interfaces to block iterative solvers
+for sparse linear and eigenvalue problems. In contrast to other libraries we
+support multiple backends (e.g. Trilinos, PETSc and our own optimized kernels),
+and interfaces in multiple languages such as C, C++, Fortran 2003 and Python.
+PHIST has a clear focus on portability and hardware performance: in particular
+we support row-major storage of block vectors and using GPUs (via the ghost
+library or Trilinos/Tpetra).
+"""
+
+    homepage = "https://bitbucket.org/essex/phist/"
+
+    version('develop',
+            git='https://bitbucket.org/essex/phist/phist.git', branch='devel')
+    version('master',
+            git='https://bitbucket.org/essex/phist/phist.git', branch='master')
+    version('1.4.3', 'af3300378d4282366d148e38c3a3199a',
+            url='https://bitbucket.org/essex/phist/get/phist-1.4.3.tar.gz')
```

Comment:
Can you move this `url` up below `homepage`? I've seen weird bugs in Spack with packages that don't have a `url` attribute set.